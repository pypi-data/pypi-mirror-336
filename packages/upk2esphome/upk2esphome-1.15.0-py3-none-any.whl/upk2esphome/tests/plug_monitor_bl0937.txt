{"gw_bi":{"uuid":"","psk_key":"","auth_key":"","ap_ssid":"SmartLife","ap_passwd":null,"country_code":"CN","bt_mac":null,"bt_hid":null,"prod_test":false},"user_param_key":{"bt1_lv":0,"bt1_pin":10,"bt1_type":0,"ch1_stat":2,"ch_cddpid1":9,"ch_dpid1":1,"ch_num":1,"chip_type":0,"crc":74,"ele_pin":7,"jv":"1.1.5","lose_vol":85,"module":"WB2S","net_trig":1,"netled1_lv":0,"netled1_pin":6,"netn_led":2,"nety_led":2,"over_cur":17000,"over_vol":270,"reset_t":3,"resistor":1,"rl1_lv":1,"rl1_pin":26,"rl1_type":0,"sel_pin_lv":1,"sel_pin_pin":24,"vi_pin":8,"vol_def":0},"gw_wsm":{"nc_tp":1,"ssid":null,"passwd":null,"md":0,"random":0,"wfb64":1,"stat":1,"token":null,"region":null,"reg_key":null,"dns_prio":0},"gw_di":{"abi":0,"id":"bfb05a09277a67ea71si8l","swv":"1.3.2","bv":"40.00","pv":"2.2","lpv":"3.3","pk":"keym9qkuywghyrvs","firmk":"keym9qkuywghyrvs","cadv":"1.0.2","cdv":"1.0.0","dev_swv":"1.3.2","s_id":"000002otjf","dtp":0,"sync":0,"attr_num":0,"mst_tp_0":0,"mst_ver_0":null,"mst_tp_1":0,"mst_ver_1":null,"mst_tp_2":0,"mst_ver_2":null,"mst_tp_3":0,"mst_ver_3":null,"dminfo_name":null,"dminfo_code":null,"dminfo_report_code":null,"dminfo_sn":null},"is_stride":0,"timer_arr":{"lastFetchTime":0,"cnt":0},"tls_ca_cnt":0,"gw_ai":{"key":"f036e56951c1dbb5","lckey":"3b8f837aa200f866","h_url":"http://a.tuyaeu.com/d.json","h_ip":"54.93.90.113","hs_url":null,"hs_ip":null,"hs_psk":"https://a3.tuyaeu.com/d.json","hs_psk_ip":"18.185.182.159","mqs_url":null,"mqs_ip":null,"mq_url":"m2.tuyaeu.com:1883","mq_ip":"35.156.42.116","ai_sp":null,"mq_psk":"m2.tuyaeu.com:8886","mq_psk_ip":"35.156.42.116","time_z":"+01:00","s_time_z":"[[1648342800,1667091600],[1679792400,1698541200]]","wx_app_id":null,"wx_uuid":null,"dy_tls_m":0,"cloud_cap":1025},"000002otjf":[{"type":"obj","mode":"rw","property":{"type":"bool"},"id":1},{"mode":"rw","property":{"min":0,"max":86400,"scale":0,"step":1,"type":"value"},"id":9,"type":"obj"},{"mode":"ro","property":{"min":0,"max":50000,"scale":3,"step":100,"type":"value"},"id":17,"type":"obj"},{"mode":"ro","property":{"min":0,"max":30000,"scale":0,"step":1,"type":"value"},"id":18,"type":"obj"},{"mode":"ro","property":{"min":0,"max":80000,"scale":1,"step":1,"type":"value"},"id":19,"type":"obj"},{"mode":"ro","property":{"min":0,"max":5000,"scale":1,"step":1,"type":"value"},"id":20,"type":"obj"},{"mode":"ro","property":{"min":0,"max":5,"scale":0,"step":1,"type":"value"},"id":21,"type":"obj"},{"mode":"ro","property":{"min":0,"max":100000000,"scale":0,"step":1,"type":"value"},"id":22,"type":"obj"},{"mode":"ro","property":{"min":0,"max":100000000,"scale":0,"step":1,"type":"value"},"id":23,"type":"obj"},{"mode":"ro","property":{"min":0,"max":1000000000,"scale":0,"step":1,"type":"value"},"id":24,"type":"obj"},{"mode":"ro","property":{"min":0,"max":1000000,"scale":0,"step":1,"type":"value"},"id":25,"type":"obj"},{"mode":"ro","property":{"type":"bitmap","maxlen":6},"id":26,"type":"obj"}],"em_sys_env":"BK7231S_2M"}
