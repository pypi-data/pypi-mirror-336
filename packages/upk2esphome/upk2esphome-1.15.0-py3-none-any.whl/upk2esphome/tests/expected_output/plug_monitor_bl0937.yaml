binary_sensor:
  - platform: gpio
    id: binary_switch_1
    pin:
      number: P10
      inverted: true
      mode: INPUT_PULLUP
    on_press:
      then:
        - switch.toggle: switch_1

switch:
  - platform: gpio
    id: switch_1
    name: Relay 1
    pin: P26

status_led:
  pin:
    number: P6
    inverted: true

sensor:
  - platform: hlw8012
    model: BL0937
    cf_pin:
      number: P7
      inverted: true
    cf1_pin:
      number: P8
      inverted: true
    sel_pin:
      number: P24
      inverted: true
    current:
      name: BL0937 Current
    voltage:
      name: BL0937 Voltage
    power:
      name: BL0937 Power
    energy:
      name: BL0937 Energy
    voltage_divider: 1600
    current_resistor: 0.001 ohm
